src/wp-includes/blocks.php

```diff
@@ -1239,8 +1281,12 @@ function get_query_pagination_arrow( $block, $is_next ) {
 function _wp_multiple_block_styles( $metadata ) {
 	foreach ( array( 'style', 'editorStyle' ) as $key ) {
 		if ( ! empty( $metadata[ $key ] ) && is_array( $metadata[ $key ] ) ) {
-			$default_style = array_shift( $metadata[ $key ] );
 			foreach ( $metadata[ $key ] as $handle ) {
+				// Do not enqueue style arrays such as { "color": { "text": "#fff" } }.
```

Comment:
So this changes the behavior of both the `editorStyle` and `style`, but we have only changed handling for `style` in the rest of the codebase.

@aristath, can we get rid of this `_wp_multiple_block_styles` callback if we add native handling for an array in `WP_Block_Type`?

Reply:
Would make perfect sense, yes 😄

Reply:
Done!